libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -467,6 +467,7 @@ def add_version_header(tc):
                 # "c++20": 202110 Not implemented P2372R3 Fixing locale handling in chrono formatters
                 "c++20": 202106,
                 # "c++23": 202207, Not implemented P2419R2 Clarify handling of encodings in localized formatting of chrono types
+                # "c++26": 202306, # P2637R3 Member Visit
```

Comment:
```suggestion
                # "c++26": 202306, P2637R3 Member Visit (implemented)
```
Just to make it easier to keep track of the FTM.